src/ovp_pipeline/unified_pipeline_enhanced.py

```
@@ -2565,6 +2574,101 @@ def step_evergreen(self, recent_days: int = 7, dry_run: bool = False, quality_sc
+                self.vault_dir, llm_call=llm_call
+            )
+
+            absorb_result = self.step_results.get("absorb", {})
+            absorb_files = absorb_result.get("processed_files", [])
+
+            if not absorb_files:
+                areas_dir = self.vault_dir / "20-Areas"
+                if areas_dir.exists():
+                    from datetime import datetime, timedelta
+
+                    cutoff = datetime.now() - timedelta(days=7)
```

Ruff (0.15.12)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)